price_pusher/src/injective.ts

```diff
@@ -103,10 +122,126 @@ export class InjectivePriceListener implements PriceListener {
 }
 
 export class InjectivePricePusher implements ChainPricePusher {
+  private wallet: PrivateKey;
+  constructor(
+    private cwPriceServiceConnection: CwPriceServiceConnection,
+    private pythContract: string,
+    private grpcEndpoint: string,
+    mnemonic: string
+  ) {
+    this.wallet = PrivateKey.fromMnemonic(mnemonic);
+  }
+
+  private injectiveAddress(): string {
+    return this.wallet.toBech32();
+  }
+
+  private async signAndBroadcastMsg(
```

Comment:
It is a bit surprising to me that injective does not have a proper method for doing it.